samples/drivers/i2s/i2s_codec/Kconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config I2S_INIT_BUFFERS
+	int "Initial count of audio data blocks"
+	default 2
+	help
+	  Controls the initial count of audio data blocks, which are (optionally)
+	  filled by data from the DMIC peripheral and played back by the I2S
+	  output perihperal.
+
+config SAMPLE_FREQ
+	int "Sample rate"
+	default 48000
+	help
+	  Sample frequency of the system.
```

Comment:
please be more specific. what is the "system"?

Reply:
Well, the microcontroller along with the codec. The sample frequency is the only one configured. The codec is configured for that rate and samples are processed at that rate. Can't think of a better designation, honestly.